### PR TITLE
Fixed display text and popover for licenses and attribution

### PR DESCRIPTION
**Pull request by qtomlinson**
1. Fixed popover hidden
The popover is not visible because styling from bootstrap v5.1.1 conflicted with reactjs bootstrap popover (v0.32.3, Bootstrap 3) currently used.  A visible popover in clearlyDefined UI is styled with "fade in popover left" classes. The following style from bootstrap v5.1.1 hides the popover because the popover does not have "show" class:
.fade:not(.show) {
    opacity: 0;
}
Turn off animation for popover so the conflicting styling will not be effective.

2.  The tool list is displayed in its entirety in "Tools" section.  Popover contains only duplicate information and seems redundant.  Removed ListDataRenderer to avoid pop up.

3. Fixed display text.

Task: https://github.com/clearlydefined/website/issues/950

### CODE REVIEW

**Comment by qtomlinson** [comment]
@disulliv @mpcen ready for review.

File: src/styles/_Popover.scss [hunk 1]
```diff
@@ -51,6 +51,7 @@
   text-overflow: ellipsis;
   width: 100%;
   display: block;
+  cursor: pointer;
```
Comment (disulliv) [inline src/styles/_Popover.scss:54]: any chance you could update the styling to match the styles of the website? Font, etc.

**Review by disulliv — CHANGES_REQUESTED** [review]
Looks good to me besides the styling, thanks!

**Review by mpcen — APPROVED** [review]

**Comment by qtomlinson** [comment]
See https://github.com/clearlydefined/website/pull/979/commits/a15a45232443f214a8d610fc35f097edeb688278

**Review by disulliv — APPROVED** [review]

**Pull request merged**